build-scratch

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
-SHA=`git rev-parse HEAD`
-docker build -t $SHA . &&
-    docker run --rm --entrypoint tar $SHA -c /usr/bin/ct Dockerfile.build-scratch | docker build -f Dockerfile.build-scratch --tag coreos/ct -
+
+set -e
+VERSION=$(./git-version)
```

Comment:
Ah, I was running it in another repo. Seems this fails if you don't have any tags yet. I suppose you don't care about that case?